RockPaperScissors/RockPaperScissors.cs

```diff
@@ -6,20 +6,138 @@ class Program
+            string humanName = Console.ReadLine();
+            Console.WriteLine("\nHello, {0}!\n", humanName);
+            System.Threading.Thread.Sleep(1000);
+            Console.WriteLine("What would you like to name your opponent?");
+            string compName = Console.ReadLine();
+
+            Console.WriteLine("Okay, Let's play! Type EXIT at any time to quit. Type carefully as an invalid inmput will result in computer winning!");
+            System.Threading.Thread.Sleep(1000);
+            
+            //Game loop
+            while (playing)
+            {
+                round += 1;
+
+                //get human player's choice.
+                Console.WriteLine("\nRound {0}\nChoose your hand: \"rock\", \"paper\" or \"scissors\":", round);
+                string hand1 = Console.ReadLine().ToLower();
+
+                //see if Exit.
+                if (hand1 == "exit" || hand1 == "quit") {
+                    playing = false;
+                    return;
+                }
+
+                //choose a random number from 1-3 for the computer.
+                Random rndX = new Random();
+                int compChoice = rndX.Next(1,3);
+
+                //determine the computer's hand based on our random number.
+                string hand2 = null;
+                switch (compChoice)
+                {
+                    case 1:
+                        hand2 = "rock";
+                        break;
+                    case 2:
+                        hand2 = "paper";
+                        break;
+                    case 3:
+                        hand2 = "scissors";
+                        break;
+                }
+                //Tell us what computer chose.
+                Console.WriteLine("\n{0} chose {1}!\n", compName, hand2);
+                System.Threading.Thread.Sleep(1000);
+
+
+                //declare winner.
+                int winner = CompareHands(hand1, hand2);
+
+                switch (winner)
+                {
+                    case 0: //tie
+                        Console.WriteLine("It's a tie! No points awarded this round.");
+                        break;
+                    case 1: //human wins
+                        Console.WriteLine("You win!");
+                        score1 += 1;
+                        break;
+                    default: //computer wins
+                        Console.WriteLine("{0} wins!", compName);
+                        score2 += 1;
+                        break;
+                }
+                System.Threading.Thread.Sleep(1000);
+
+                //display score.
+                Console.WriteLine("\nScoreboard:\n{0}: {1}\n{2}: {3}", humanName, score1, compName, score2);
+                System.Threading.Thread.Sleep(1000);
+            }
         }
         
-        public static string CompareHands(string hand1, string hand2)
+        public static int CompareHands(string hand1, string hand2)
```

Comment:
Some comments explaining what this method does would be nice.
For example:
This method returns 0 if it is a tie, 1 if hand1 is the winner, or 2 if hand 2 is the winner.